app/src/test/java/com/ikarabulut/BoardTest.java

```diff
@@ -69,4 +72,137 @@ void setRandomGrid() {
         assertFalse(isDeadBoard);
     }
 
-}
```

Comment:
Note: When you see this symbol in a code diff, it means your text editor did not add a newline character `\n` at the end of the file.  This can make files harder to process by automated analysis tools. So, it is a best practice to configure your text editor to add them for you when you edit a file.  Normally a UNIX editor will do this for you, like [vim does](https://vi.stackexchange.com/questions/3298/how-to-make-vim-automatically-add-a-newline-to-the-end-of-a-file).  But not all editors do this and need to be configured to do so.

Reply:
Good to know. I guess I should have looked through the file myself to even catch those. I updated my IntelliJ settings to add the newline. On my local file it looks like it added the newlines but git is saying I am still working a clean branch so I can't push those changes. Maybe when I have my next commit it will update